query-connector/src/app/database-service.ts

```diff
@@ -656,8 +656,9 @@ export async function getCustomQueries(): Promise<CustomUserQuery[]> {
       LEFT JOIN valueset_to_concept vtc ON vtc.valueset_id = v.id
       LEFT JOIN concepts c ON c.id = vtc.concept_id
     ) vc ON vc.valueset_id = qtv.valueset_id
-    WHERE q.author = 'DIBBs';
+    WHERE     q.query_name IN ('Gonorrhea (disorder)', 'Newborn Screening');
```

Comment:
I know we are going to remove eventually so that all queries are displayed when an admin logs in so could we add a `TODO` and remove the commented out line below?

Reply:
closing the loop, adding follow-up ticket to track needed refactor here: https://linear.app/skylight-cdc/issue/QUE-85/update-database-servicests-getcustomqueries